tests/e2e.test.ts

```diff
@@ -0,0 +1,859 @@
+import grpc = require('grpc');
+import {
+    insert_grpc,
+    search_grpc,
+    update_grpc,
+    upsert_grpc,
+    remove_grpc,
+    object_grpc,
+} from '../src/vald/v1/vald';
+import { agent_grpc } from '../src/vald/v1/agent/core';
+import { payload } from '../src/vald/v1/payload';
+import { InsertClient } from '../src/vald/v1/vald/insert_grpc_pb';
+import { RemoveClient } from '../src/vald/v1/vald/remove_grpc_pb';
+import { SearchClient } from '../src/vald/v1/vald/search_grpc_pb';
+import { UpdateClient } from '../src/vald/v1/vald/update_grpc_pb';
+import { UpsertClient } from '../src/vald/v1/vald/upsert_grpc_pb';
+import { ObjectClient } from '../src/vald/v1/vald/object_grpc_pb';
+import { AgentClient } from '../src/vald/v1/agent/core/agent_grpc_pb';
+import data = require('./wordvecs1000.json');
+
+const address = 'localhost:8081';
+const connectDeadlineMs = 10000;
+
+jest.setTimeout(60000);
+
+describe('Tests for node client', () => {
+    describe('Tests for insert operations', () => {
+        let client: InsertClient;
+
+        beforeAll(done => {
+            client = new insert_grpc.InsertClient(
+                address,
+                grpc.credentials.createInsecure(),
+            );
+            client.waitForReady(Date.now() + connectDeadlineMs, (e) => {
+                if (e) {
+                    done(e);
+                } else {
+                    done();
+                }
+            });
+        });
+
+        afterAll(() => {
+            client.close();
+        });
+
+        test('Insert', done => {
+            const vec = new payload.Object.Vector();
+            vec.setId(data[0].id);
+            vec.setVectorList(data[0].vector);
+
+            const cfg = new payload.Insert.Config();
+            cfg.setSkipStrictExistCheck(true);
+
+            const req = new payload.Insert.Request();
+            req.setVector(vec);
+            req.setConfig(cfg);
+
+            client.insert(req, (err, resp) => {
+                try {
+                    expect(err).toBeFalsy();
+                    expect(resp).toBeInstanceOf(payload.Object.Location);
+                    done();
+                } catch (e) {
+                    done(e);
+                }
+            });
+        });
+
+        test('MultiInsert', done => {
+            const requests = [];
+            const cfg = new payload.Insert.Config();
+            cfg.setSkipStrictExistCheck(true);
+
+            for (let i = 1; i < 11; i++) {
+                const vec = new payload.Object.Vector();
+                vec.setId(data[i].id);
+                vec.setVectorList(data[i].vector);
+
+                const r = new payload.Insert.Request();
+                r.setVector(vec);
+                r.setConfig(cfg);
+
+                requests.push(r);
+            }
```

Comment:
```suggestion
            const vec = new payload.Object.Vector();
            const r = new payload.Insert.Request();
            for (let i = 1; i < 11; i++) {
                vec.setId(data[i].id);
                vec.setVectorList(data[i].vector);

                r.setVector(vec);
                r.setConfig(cfg);

                requests.push(r);
            }
```
you can define constant valuables out of loop.
but, I'm not sure which is the better.

Reply:
We can't do like that because an element of an array is just a reference to the passed value.
Your code will result that all the elements in the array will be same.
```
Welcome to Node.js v14.15.5.
Type ".help" for more information.
> const arr = [];
undefined
> const a = ['a', 'b', 'c'];
undefined
> const count = arr.push(a);
undefined
> console.log(arr);
[ [ 'a', 'b', 'c' ] ]
undefined
> a.push('d');
4
> arr.push(a);
2
> console.log(arr);
[ [ 'a', 'b', 'c', 'd' ], [ 'a', 'b', 'c', 'd' ] ]
```

Reply:
arr, I misread the code.
you are right.